### ISSUE TITLE
Missing 'Mapping' from 'collections' still a problem

### ISSUE THREAD

**Issue by kzembower** [FrancescoCeruti/linux-show-player#296]
**Describe the bug**
When I try to run linux-show-player on Ubuntu  22.04.3 LTS, I  get the error:

`ImportError: cannot import name 'Mapping' from 'collections' (/usr/lib/python3.10/collections/__init__.py)
`
**To Reproduce**
run linux-show-player on Ubuntu  22.04.3 LTS with  Python 3.10.12

**Full error output**
```
$ linux-show-player 
Traceback (most recent call last):
  File "/usr/bin/linux-show-player", line 3, in <module>
    from lisp.main import main
  File "/usr/lib/python3/dist-packages/lisp/main.py", line 30, in <module>
    from lisp import modules
  File "/usr/lib/python3/dist-packages/lisp/modules/__init__.py", line 23, in <module>
    from lisp.ui import elogging
  File "/usr/lib/python3/dist-packages/lisp/ui/elogging.py", line 28, in <module>
    from lisp.ui.widgets import QDetailedMessageBox
  File "/usr/lib/python3/dist-packages/lisp/ui/widgets/__init__.py", line 1, in <module>
    from .cueaction_combobox import CueActionComboBox
  File "/usr/lib/python3/dist-packages/lisp/ui/widgets/cueaction_combobox.py", line 25, in <module>
    from lisp.cues.cue import Cue, CueAction
  File "/usr/lib/python3/dist-packages/lisp/cues/cue.py", line 25, in <module>
    from lisp.core.fade_functions import FadeInType, FadeOutType
  File "/usr/lib/python3/dist-packages/lisp/core/fade_functions.py", line 32, in <module>
    from lisp.core.util import FunctionProxy
  File "/usr/lib/python3/dist-packages/lisp/core/util.py", line 22, in <module>
    from collections import Mapping
ImportError: cannot import name 'Mapping' from 'collections' (/usr/lib/python3.10/collections/__init__.py)
$
```
**Linux Show Player version:**
Unknown; can't run 'linux-show-player --version' without error.  However, apt-cache policy shows:
```
$ apt-cache policy linux-show-player
linux-show-player:
  Installed: 0.5.2-1
  Candidate: 0.5.2-1
  Version table:
 *** 0.5.2-1 500
        500 http://us.archive.ubuntu.com/ubuntu jammy/universe amd64 Packages
        500 http://us.archive.ubuntu.com/ubuntu jammy/universe i386 Packages
        100 /var/lib/dpkg/status
$ 
```
**OS Version**
```
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.3 LTS
Release:	22.04
Codename:	jammy
$
```

**Additional context**
Despite the closing of issue https://github.com/FrancescoCeruti/linux-show-player/issues/281, I still experience this problem.

Please let me know if this problem can be easily solved by switching to another version of python3 (3.11?) or another easy fix.

Thanks for looking into this issue, and a special thanks to Francesco for creating and maintaining linux-show-player. I use it to play sound cues during lectures I present.

-Kevin

**Comment by s0600204** [comment]
The fix is included with the 0.5.**3** release of Linux Show Player, so it is not surprising you're still suffering from it. It is up to the repository maintainers of Ubuntu to make this available via the Ubuntu software package repositories, not us.

There has been a [ticket open on Ubuntu's bug report system](https://bugs.launchpad.net/ubuntu/+source/linux-show-player/+bug/1987073) for over a year asking for the newer version to be provided, but no joy thus far. (If you have an UbuntuOne login, feel free to indicate on that page that the bug affects you too.)

The AppImage has its own problems, and the FlatPaks contain the upcoming `0.6` version, so if you wish to continue using Linux Show Player `0.5`, than you will need to [build and install from source](https://linux-show-player-users.readthedocs.io/en/latest/installation.html#manually).

----

Duplicate of #238, #245, #250, #261

**Comment by kzembower** [comment]
Thank you so much for your quick and complete response.

I indicated on the UbuntuOne page that this affects me to.

I'm comfortable building from source, so I'll explore that.

Thanks, again, and sorry for not noticing all the other duplicates of this issue.

-Kevin
